server/routers/admin_dashboard.py

```
@@ -149,6 +165,57 @@ async def get_recent_activities(
+                "description": description,
+                "location_name": d.location_name,
+                "severity": severity_label,
+                "severity_level": sev,
+                "extracted_at": d.extracted_at.isoformat() if d.extracted_at else None,
+                "event_time": event_time.isoformat() if event_time else None,
+                "latitude": d.latitude,
+                "longitude": d.longitude,
+            })
+        
+        return {"crises": crises}
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)